src/pages/Books.js

```diff
@@ -1,20 +1,9 @@
+import { useSelector } from 'react-redux';
 import AddBook from '../components/AddBook';
+import RemoveBook from '../components/RemoveBook';
 
 const Books = () => {
-  const books = [
-    {
-      id: 1,
-      title: 'The Lord of the Rings',
-      author: 'J. R. R. Tolkien',
-      category: 'Fantasy',
-    },
-    {
-      id: 2,
-      title: 'Among the Thugs',
-      author: 'Bill Buford',
-      category: 'Journalism',
-    },
-  ];
+  const books = useSelector((state) => state.booksReducer);
 
   return (
     <div className="book-section">
```

Comment:
**Changes Required**
- [ ] This should be a `Book Component` that take these values as props. Let us refactor it into a Book Component and just import it here passing these values as props to it.

Reply:
Good observation.